src/main/java/org/automation/pages/TransactionsPage.java

```diff
@@ -60,20 +60,20 @@ public class TransactionsPage extends BasePage {
     By pendingPayments = By.xpath("//option[@value='pending']");
     By failedPayments = By.xpath("//option[@value='failed']");
     By unverifiedPayments = By.xpath("//option[@value='unverified']");
-    By pendingPaymentIcon = By.xpath("//i[contains(@class,'fas fa-hourglass-half text-warning fs-5 me-2')]");
-    By excalamatrySign = By.xpath("//i[contains(@class,'fas fa-exclamation-circle text-danger fs-5 me-2')]");
-    By quotionmarkSign = By.xpath("//i[@class='fas fa-question-square text-info fs-5 me-2']");
-    By qrCodeSign = By.xpath("//i[contains(@class,'fa fa-qrcode me-2')]");
-    By paymentLinkField = By.xpath("//select[@name='billTemplate']");
+    public By pendingPaymentIcon = By.xpath("//i[contains(@class,'fas fa-hourglass-half text-warning fs-5 me-2')]");
+   public By excalamatrySign = By.xpath("//i[contains(@class,'fas fa-exclamation-circle text-danger fs-5 me-2')]");
+   public By quotionmarkSign = By.xpath("//i[@class='fas fa-question-square text-info fs-5 me-2']");
+   public By qrCodeSign = By.xpath("//i[contains(@class,'fa fa-qrcode me-2')]");
+   public By paymentLinkField = By.xpath("//select[@name='billTemplate']");
     By qrCodeSeletct = By.xpath("(//select[@name='billTemplate']/option)[2]");
```

Comment:
Avoid using position-based locators (e.g., [2]). These are brittle and can break with changes in the UI structure. Use attributes or text for reliable identification.